.circleci/config.yml

```diff
@@ -1198,7 +1198,7 @@ jobs:
       - run:
           name: Prepare test environment (compile tests and cache results)
           working_directory: op-acceptance-tests
-          command: go test -run=^$ ./tests/...
+          command: DEVSTACK_ORCHESTRATOR="sysext" go test -run=^$ ./tests/...
```

Comment:
Can you drop this change before merging? It should use sysgo by default.

@scharissis has prepared a separate acceptance-testing workflow for the kurtosis backend based testing.

Reply:
Sadly I can't because it breaks the kurtosis tests without it. This is part of the setup for running in kurtosis (the `-run=^$` is actually telling it not to run anything so it's really just compiling the tests, but in doing so it runs the setup code which starts all the go backend networks which fail because cannon and op-program haven't been compiled on the runner itself (only in the kurtosis devnet).

We could wait to merge @scharissis work first but getting challenger running in sysgo would make work on the proofs ATs dramatically faster and easier.

Reply:
When we use op-acceptor to run the tests it uses `sysext` ([code](https://github.com/ethereum-optimism/optimism/blob/develop/op-acceptance-tests/cmd/main.go#L179-L182))

I'm surprised about this part:
> but in doing so it runs the setup code which starts all the go backend networks which fail because cannon and op-program haven't been compiled on the runner itself (only in the kurtosis devnet)

I don't understand why it passes elsewhere (such as in other PRs/branches)? Is this issue specific to challenger?

Reply:
Yes the dependency on having cannon built is introduced by running challenger.  So we need to trigger the compile using the same backend as we actually want to use to run the tests - otherwise we need to setup the dependencies for both (but mostly spinning up all those chains is a waste of time).

Reply:
Ah, understood!

In that case, I think we should remove this step as it'll likely result in a similar mis-compatibility down the line.

https://github.com/ethereum-optimism/optimism/pull/16023

Reply:
With the precompile step now removed we don't need this change.